dev-ruby/http/http-5.0.4.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+USE_RUBY="ruby27 ruby30 ruby31"
+
+RUBY_FAKEGEM_RECIPE_TEST="rspec3"
+RUBY_FAKEGEM_EXTRADOC="CHANGES.md README.md"
+RUBY_FAKEGEM_GEMSPEC="${PN}.gemspec"
+# disabled for now.  pulls in another large dep tree for ruby31
```

Comment:
yard now have a ruby31 target. We also should not comment out things like this but wait for these packages to be ready for ruby31 before adding it here.